src/display3d.cpp

```diff
@@ -1351,6 +1357,10 @@ static void drawTiles(iView *player)
 	const Vector3f theSun = (viewMatrix * glm::vec4(getTheSun(), 0.f)).xyz();
 	pie_BeginLighting(theSun);
 
+	// Reset all lighting data
+	getCurrentLightingData().lights.clear();
```

Comment:
Since lights have a lifetime of a single frame in warzone engine, I reset the lists of light at the beginning of a frame.